### ISSUE TITLE
Obfuscate email addresses so spammers don't collect them as easily.

### ISSUE THREAD

**Issue by therealromster** [circuits/irclogger#1]
I would like to see email addresses posted in irc, for those that post them without first obfuscating them to be done when it's logged for display by the irc logger.

Something like foo at bar dash example dot com

Easy enough for a human to use should they need too. but hard for a bot.

Although the best way would be to use a small javascript code to hide it from spam bot parsers.

```
<script language=JavaScript>
<!--
document.write("foo" + "@" + "bar" + "-" + "example" + "." + "com");
//-->
</script>
```

**Comment by prologic** [comment]
Absolutely great idea. and Trivial to add too by creating a "filter" component to intercept `log` messages before they get stored on disk.

Alternatively this would be done on the fly in a new Web UI / Log Viewer. The former approach would mean you could not get at the real email addresses at all if you obfuscated it too much.

**Comment by pdurbin** [comment]
I'd probably favor persisting the data "as is" and having email addresses filtered out by some UI or script.

**Comment by therealromster** [comment]
Perhaps even have a show email links with some kapacha picture if you wanted to go that far.

**Comment by prologic** [comment]
Agreed. Right now  irclogger uses [irclog2html](https://pypi.python.org/pypi/irclog2html) -- So either this would have to be extended or used in conjunction with some static site generation tools or a complete Web UI written (_in `circuits.web` ofc!_) from the ground up.

**Comment by prologic** [comment]
This is what the `update` scripts looks like right now:

``` #!bash
#!/bin/bash

export PATH="$HOME/bin:$PATH"

for dir in *; do
    if [ -d $dir ]; then
        logs2html -t "IRC Logs for $dir " -p "IRC Logs for $dir " $dir
    fi
done
```

which is run via crond:

```
*/5 * * * * cd $HOME/irclogger/tmp/irclogs && ./update
```

All this could be improved and brought back into the daemon itself really.
